app/src/main/java/org/mozilla/fenix/collections/CreateCollectionFragment.kt

```diff
@@ -64,7 +62,12 @@ class CreateCollectionFragment : DialogFragment(), CoroutineScope {
                 this,
                 CollectionCreationViewModel::class.java
             ) {
-                CollectionCreationViewModel(CollectionCreationState(tabs, selectedTabs, step, tabCollections))
+                CollectionCreationViewModel(CollectionCreationState(
```

Comment:
You don't have to do this anymore, if the viewModel exists in the store it will use it :)